### DISCUSSION TITLE
Adding Account to Project

### DISCUSSION THREAD

**Discussion by my-code-AL** [apache/cloudstack#9148] [Q&A] [answered]
Hello All,

I have a few questions regarding project initialization in Apache CloudStack. Specifically, I would like to understand the relationship between domains, users, and projects.

Domain and User Creation: Is it necessary to create an admin account under a specific domain and then create users under this same domain, and are the domains registered somewhere?

Adding Users to a Project: Once a project is created, how are users added to it? Are there specific permissions or roles that need to be assigned?

Any insights or guidance on these topics would be greatly appreciated.

Thank you!

**Comment by shwstppr** [comment, marked as answer]
@my-code-AL 
For domains and users: it is not necessary to create domain admin account for each domain. You can create new domains and accounts under the domain as a root admin itself. I'm not sure what you mean by domains registered somewhere. CloudStack keeps track of domains as with other entities in its database. You can track domains for your env with listDomains API or Domains section in the UI. It is an organisational unit which can help track and limit usage.

Adding Users to a Project: After creating a project you can use `addAccountToProject` API or use the UI,
![Screenshot from 2024-05-30 12-37-39](https://github.com/apache/cloudstack/assets/153340/ad180cb4-6eb2-48fd-987c-2697e5cb33fc)

You can add existing account or user to the project.
![Screenshot from 2024-05-30 12-38-01](https://github.com/apache/cloudstack/assets/153340/854522c7-86ef-4629-897b-88f746c3b3fa)

**Reply by my-code-AL** [reply]
Thank you for the clarification!
